api/app.js

```diff
@@ -1,21 +1,26 @@
 const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
- 
-const mongoose = require('./db/mongoose');
-
+require('dotenv').config();
 
+const port = process.env.PORT || 3000;
```

Comment:
If you do make other commits and merge this into deployment, using this will allow it to run on heroku, seeing as heroku provides a port and gives that info through an env variable, if not provided then just fallback to port 3000

Reply:
**REVERTED**